qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -177,6 +180,30 @@ public void test70BindMountCustomPathConfAndJvmOptions() throws Exception {
         assertThat(nodesResponse, containsString("\"using_compressed_ordinary_object_pointers\":\"false\""));
     }
 
+    /**
+     * Check that the default config can be overridden using a bind mount, and that env vars are respected
+     */
+    public void test71BindMountCustomPathWithDifferentUID() throws Exception {
+        copyFromContainer(installation.config("elasticsearch.yml"), tempDir.resolve("elasticsearch.yml"));
+
+        final Path tempDataDir = mktempDir(getTempDir());
```

Comment:
`tempDir` in `DockerTests` is recreated for each test, so you don't need to create it yourself.

Reply:
Or is this due to the UI / GUI hacking?

Reply:
Good point. I removed it in d4d658a3f98c363d0b2b850f41eac4d0a4d3feaa